CRM/Upgrade/Incremental/sql/4.6.3.mysql.tpl

```diff
@@ -2,4 +2,14 @@
 -- CRM-16307 fix CRM-15578 typo - Require access CiviMail permission for A/B Testing feature
 UPDATE civicrm_navigation
 SET permission = 'access CiviMail', permission_operator = ''
-WHERE name = 'New A/B Test' OR name = 'Manage A/B Tests';
\ No newline at end of file
+WHERE name = 'New A/B Test' OR name = 'Manage A/B Tests';
+
+--CRM-16391: Rename 'Type' to 'Financial Type' in Contribution and Pledge batch entry profiles
+UPDATE civicrm_uf_field
+SET label = 'Financial Type'
```

Comment:
This script will fail incase of multi-lingual upgrade. Don't we need ts() for 'Financial Type'?